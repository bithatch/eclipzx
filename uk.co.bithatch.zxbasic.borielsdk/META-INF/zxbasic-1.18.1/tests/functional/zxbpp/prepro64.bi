
#define a <pos.bas>

#include a
