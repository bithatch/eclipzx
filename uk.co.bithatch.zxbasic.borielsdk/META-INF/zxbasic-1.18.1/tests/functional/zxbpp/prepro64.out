#line 1 "prepro64.bi"

#line 3 "prepro64.bi"

#line 1 "/zxbasic/src/lib/arch/zx48k/stdlib/pos.bas"













#line 15 "/zxbasic/src/lib/arch/zx48k/stdlib/pos.bas"

#pragma push(case_insensitive)
#pragma case_insensitive = true









function FASTCALL pos as ubyte
    asm
    push namespace core
    PROC

    call __LOAD_S_POSN
    ld <pos.bas>, e

    ENDP
    pop namespace
    end asm
end function

#pragma pop(case_insensitive)
#require "sposn.asm"


#line 45 "/zxbasic/src/lib/arch/zx48k/stdlib/pos.bas"
#line 5 "prepro64.bi"
